source/develop/release-management/features/network/detailed-host-network-qos.html.md

```diff
@@ -177,7 +177,7 @@ Since QoS API has been added in oVirt 3.5, it will be possible to expose Host Ne
 
 <b>Slightly out of date - should be similar, but other values will appear in the dialogs. A new category should appear under the DC/QoS subtab.</b>
 
-![ thumb | right](NetworkDialogQos.png  "fig: thumb | right") ![ thumb | right](InterfaceQosOverride.png  "fig: thumb | right")
+![ thumb | right](/images/wiki/NetworkDialogQos.png) ![ thumb | right](/images/wiki/InterfaceQosOverride.png)
```

Comment:
This is editorial changes. Unless there is a clear way to automate replacement this is outside the scope of this PR.